client/lib/tree-convert/test/fixtures.js

```diff
@@ -1,60 +1,58 @@
-module.exports = {
-	menusFlat: {
-		menus : [
-			{
-				id: 1,
-				name: 'Menu 1',
-				items: [
-					{ id: 1, name: 'Home', order: 1, parent: 0 },
-					{ id: 2, name: 'About us', order: 3, parent: 0 },
-					{ id: 3, name: 'Products', order: 2, parent: 0 },
-					{ id: 4, name: 'Slippers', order: 2, parent: 3 },
-					{ id: 5, name: 'Socks', order: 1, parent: 3 },
-					{ id: 6, name: 'Stripy socks', order: 1, parent: 5 },
-					{ id: 7, name: '80s socks', order: 2, parent: 5 },
-					{ id: 8, name: 'Location', order: 1, parent: 2 },
-					{ id: 9, name: 'Space invader designs', order: 1, parent: 7 }
-				],
-				locations: [ 'primary' ]
-			},
-			{
-				id: 2,
-				name: 'Menu 2 &amp;c.',
-				items: [
-					{ id: 1, name: 'Item 21 &amp; counting', order: 1 },
-					{ id: 2, name: 'Item 22', order: 2 },
-					{ id: 3, name: 'Item 23', order: 3 }
-				],
-				locations: null
-			},
-			{
-				id: 3,
-				name: 'Menu 3',
-				description: 'The third menu is dull &amp; boring',
-				items: [
-					{ id: 1, name: 'Item 31', order: 1 },
-					{ id: 2, name: 'Item 32', order: 2 },
-					{ id: 3, name: 'Item 33', order: 3 }
-				],
-				locations: [ 'secondary' ]
-			}
-		],
-		locations : [
-			{ name: 'primary', description: 'Top &amp; Primary Menu' },
-			{ name: 'secondary', description: 'Secondary menu in left sidebar' },
-			{ name: 'social&amp;lite', description: 'Social Menu' }
-		]
-	},
+export const menusFlat = {
```

Comment:
We should move this file to subfolder. New name should be like:
`client/lib/tree-convert/test/fixtures/index.js`

At the moment we use `glob` to find all test files inside `test` folders using this pattern `client/**/test/*.js` (see https://github.com/Automattic/wp-calypso/blob/master/circle.yml#L21). Circle CI tries to distribute all test files found across all containers (4 at the moment). That's why we should avoid putting all non-test files directly inside `test` folder to maintain nice test distribution and avoid loading unnecessary files.